test/sys_tests/test_sys_cfdistribution.py

```diff
@@ -22,6 +27,8 @@ def main():
         CFOriginsConfig(
             domain_name='amazonia-myStackap-LXYP1MFWT9UC-145363293.ap-southeast-2.elb.amazonaws.com',
```

Comment:
Sorry I should have picked this up in your previous PR rather than this one but I hadn't noticed it...

Just a question. Can this use R53 instead of the elb dns? Also does it need to point to the web app or could we mock something?

I guess the question is: do we want our system tests to just create some cloud formation or do we want them to create cloud formation that can prop up an entire stack without a rollback?